services/intersection-api/README.md

```diff
@@ -0,0 +1,129 @@
+# CVManager Intersection API
```

Comment:
It looks like we have a fair number of tests in here we can run, however they don't seem to be mentioned. Can we get a section on testing to show how to run those?